src/LazyLoad.jsx

```diff
@@ -155,6 +155,11 @@ export default class LazyLoad extends Component {
     return (
       <div className={elClasses} style={elStyles}>
         {img}
+        {noScript ? (<noscript
+          dangerouslySetInnerHTML={{
```

Comment:
why to use dangerouslyHTML? Ideally it should be normal Image tag with all props except the lazy load ones.